Scripts/Debug/Yotam LCA Model.py

```diff
@@ -165,26 +163,27 @@ def get_trained_network(bipartite_graph, num_features=3, num_hidden=200, epochs=
 	mnet.add_projection(projection=pho, sender=hl, receiver=ol)
 
 	# Train network
-	mnet.run(
+	mnet.learn(
 	    inputs=input_set,
 	    minibatch_size=1,
-	    bin_execute=True
+	    bin_execute=True,
+		patience=patience,
+        min_delta=min_delt,
```

Comment:
the indentation looks weird here

Reply:
Seems to be caused due to github showing tabs vs spaces differently; I'll fix by converting all tabs to 4 spaces